core/src/main/java/de/danoeh/antennapod/core/widget/WidgetUpdater.java

```diff
@@ -83,10 +86,20 @@ public static void updateWidget(Context context, WidgetState widgetState) {
             views.setOnClickPendingIntent(R.id.imgvCover, startMediaPlayer);
             views.setOnClickPendingIntent(R.id.butPlaybackSpeed, startPlaybackSpeedDialog);
 
+            RequestOptions options = new RequestOptions()
+                    .dontAnimate()
+                    .transform(
+                            new FitCenter(),
+                            new RoundedCorners(
+                                    context.getResources().getDimensionPixelSize(R.dimen.widget_inner_radius)
+                            )
+                    );
```

Comment:
Something like this would be a lot more compact:

```suggestion
            int radius = context.getResources().getDimensionPixelSize(R.dimen.widget_inner_radius);
            RequestOptions options = new RequestOptions()
                    .dontAnimate()
                    .transform(new FitCenter(), new RoundedCorners(radius));
```

(but let's first decide if we even use the dimen)

Reply:
IMO it's nicer to use a dimen for this, even if it's 4dp on all versions. This dimen can be re-used for all other rounded cover corners and avoids this magical `4` in the code.

Reply:
I picked your code.